docs/Adding_new_distros.md

```diff
@@ -1,12 +1,12 @@
 # Adding new distributions to the tool
 
-This documents details the steps to be performed in order to add a distribution support to PDS tool. 
+This documents details the steps to be performed in order to add a distribution support to Software Discovery tool.
 
 _**General Notes:**_ 	
 
  * _A directory `/<DATA_FILE_LOCATION>/` defined in Step 6 of [Installation](Installation.md) document._
 
- * _A directory `/<PDS_BASE>/` defined in Step 6 of [Installation](Installation.md) document._
+ * _A directory `/<TOOL_BASE>/` defined in Step 6 of [Installation](Installation.md) document._
```

Comment:
Upon closer review, maybe leave this one until the config.py changes are made? Since this directory is referenced in that file:

src/config/config.py:PDS_BASE = '/opt/software-discovery-tool'
src/config/config.py:DATA_FILE_LOCATION = '%s/distro_data' % PDS_BASE
src/config/config.py:LOG_FILE_LOCATION = '%s/log/pds.log' % PDS_BASE